overlord/install/install.go

```diff
@@ -356,7 +356,7 @@ func encryptionAvailabilityCheck(model *asserts.Model, tpmMode secboot.TPMProvis
 	return "", nil, nil
 }
 
-var preinstallCheckSupported = func(model *asserts.Model) (bool, error) {
+func PreinstallCheckSupported(model *asserts.Model) (bool, error) {
 	//TODO:FDEM: This temporary fallback must be removed before release of snapd 2.71
 	if osutil.GetenvBool("SNAPD_DISABLE_PREINSTALL_CHECK") {
 		logger.Noticef(`preinstall check disabled by environment variable "SNAPD_DISABLE_PREINSTALL_CHECK"`)
```

Comment:
would it make sense to extract a helper without the environment variable check? since the recovery key restrictions is not something we want users to be able to bypass